code/modules/mob/living/simple_animal/friendly/robot_customer.dm

```diff
@@ -1,8 +1,8 @@
 ///Robot customers
 /mob/living/simple_animal/robot_customer
 	name = "space-tourist bot"
-	maxHealth = 50000000 //go fuck yourself
-	health = 50000000
+	maxHealth = 1000 //go fuck yourself
+	health = 1000
```

Comment:
```suggestion
	maxHealth = 100
	health = 100
```

Reply:
then won't they just get murdered for shits and giggles, which is annoying for people trying to actually use them

Reply:
I think that the unusually large amount of health would actually encourage people to try to murder them, but hey, maybe I'm wrong.

If someone intentionally murders your tourist bots, I think that counts as them sabotaging your workplace, which should be a valid reason to dunk on them.

Reply:
They respawn at least one per minute